nilearn/plotting/_utils.py

```diff
@@ -13,11 +13,9 @@
 
 def engine_warning(engine):
     message = (
-        (
-            f"'{engine}' is not installed. To be able to use '{engine}' as "
-            "plotting engine for 'nilearn.plotting' package:\n"
-            " pip install 'nilearn[plotting]'"
-        ),
+        f"'{engine}' is not installed. To be able to use '{engine}' as "
+        "plotting engine for 'nilearn.plotting' package:\n"
+        " pip install 'nilearn[plotting]'"
```

Comment:
that one is my fault: I changed it when changing the stacklevel of the warnings but had not tested it properly

it also creates issues on main, that should therefore be fixed by this PR